src/modules/utility/models/User.ts

```diff
@@ -5,19 +5,19 @@ interface UserInterface {
   Username?: string;
   email?: string;
   playerName?: string;
-  SteamID?: number;
+  SteamID?: string;
   XboxID?: string;
-  DiscordID: number;
+  DiscordID: string;
 }
 
 class User extends Model<UserInterface, UserInterface> implements UserInterface {
   id?: number;
   Username?: string;
   email?: string;
   playerName?: string;
```

Comment:
Also for consistency, try not to mix the camelCase and PascalCase in the same model. ;-) Sorry I'm a pain in the ass